types/marshal/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for marshal 0.5
+// Project: https://github.com/clayzermk1/node-marshal
+// Definitions by: Jamie Magee <https://github.com/JamieMagee>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export default class Marshal {
```

Comment:
```suggestion
declare class Marshal {
```
https://unpkg.com/browse/marshal@0.5.2/lib/marshal.js
old style module export with Function construtor, IMO